tests/Microsoft.Identity.Test.Unit/WwwAuthenticateParametersTests.cs

```diff
@@ -56,6 +59,86 @@ public void CreateWwwAuthenticateResponse(string resource, string authorizationU
             Assert.IsNull(authParams.Error);
         }
 
+        [TestMethod]
+        [DataRow("AuthScheme", "" )]
+        [DataRow("AuthScheme", "realm = someRealm")]
+        [DataRow("AuthScheme", "token68")]
+        [DataRow("AuthScheme", "realm = someRealm token68")]
+        [DataRow("AuthScheme", "auth-param1=token1, auth-param2=token2, auth-param3=token3")]
+        [DataRow("AuthScheme", "realm = someRealm auth-param1=token1, auth-param2=token2, auth-param3=token3")]

```

Comment:
This is wrong: 

<img width="521" alt="image" src="https://user-images.githubusercontent.com/12273384/204033742-45a31d59-70ab-44dd-a6be-42eed13f1dfe.png">